flutter-studio/src/io/flutter/module/FlutterDescriptionProvider.java

```diff
@@ -52,17 +52,19 @@
     }
     if (projectHasFlutter) {
       // Makes no sense to add Flutter templates to Android projects...
-      res.add(new FlutterApplicationGalleryEntry(sharedModel));
+      if (isNewProject) {
+        res.add(new FlutterApplicationGalleryEntry(sharedModel));
+      }
       res.add(new FlutterPluginGalleryEntry(sharedModel));
       res.add(new FlutterPackageGalleryEntry(sharedModel));
-      if (System.getProperty("flutter.experimental.modules", null) != null) {
+      if (isNewProject) {
         res.add(new FlutterModuleGalleryEntry(sharedModel));
       }
     }
-    // TODO(messick): Create a new module type that will import an existing Flutter module into an Android app.
-    // It needs to edit /settings.gradle and /app/build.gradle as described in the add2app doc.
-    // The Project Structure editor will need to be extended to 'include' Flutter plugins for Flutter modules;
-    // without that users will have to edit /settings.gradle to import a Flutter plugin (which is true today).
+    else {
```

Comment:
The else branch here looks like the one for `!projectHasFlutter`, but the comment below says ` isNewProject == false`

Reply:
This is the block that adds items to the table shown when File>New>New Module is selected for an Android app project -- not Flutter. The only applicable item is "Import Flutter Module" and it isn't used anywhere else.